JesusFilm/core
# fix: Inline Editing - Edit Fab

# Description

Make the Editor fab support Edit and Save as well as Add Mode. 
Clicking a block should change the fab to "Edit" mode. 
Clicking on "Edit" should change it to Save mode.(displays "Done")

[Basecamp](https://3.basecamp.com/3105655/buckets/26244169/todos/4689856773)

# How should this PR be QA Tested?

- [ ] PR Checks
- [ ] On storybook or locally, when a card is displayed:
Should be "Add". 
Select a block - should be "Edit". 
Click on Edit fab - should show "Done". 
Clicking done takes you back to "Edit". 
Clicking the card / out of the card takes you back to "Add"

# Checklist
